File: PCAxis.Excel.Web.Controls/ExcelSerializerCreator.cs
using PCAxis.Paxiom;

namespace PCAxis.Excel.Web.Controls
{
    public class ExcelSerializerCreator : PCAxis.Web.Core.ISerializerCreator
    {

        public Paxiom.IPXModelStreamSerializer Create(string fileInfo)
        {
            PCAxis.Serializers.XlsxSerializer ser;
            ser = new PCAxis.Serializers.XlsxSerializer();

            if (fileInfo.Equals("FileTypeExcelXDoubleColumn"))
            {
                ser.DoubleColumn = DoubleColumnType.AlwaysDoubleColumns;
            }
            else
            {
                //Get doublecolumn from application setting
                ser.DoubleColumn = Settings.Files.DoubleColumnFile;
            }

            //    'Get information level from application setting
            ser.InformationLevel = Settings.Files.CompleteInfoFile;

            return ser;
        }
    }
}

File: PCAxis.Web.Controls/FileTypes/Excel/ExcelSerializerCreator.vb
Imports PCAxis.Paxiom

Public Class ExcelSerializerCreator
    Implements PCAxis.Web.Core.ISerializerCreator

    Public Function Create(fileInfo As String) As Paxiom.IPXModelStreamSerializer Implements Core.ISerializerCreator.Create
        Dim excelSerializer As New ExcelFileSerializer

        If fileInfo.Equals(Plugins.FileFormats.XLS_DOUBLE_COLUMN) Then
            excelSerializer.DoubleColumn = DoubleColumnType.AlwaysDoubleColumns
        Else
            'Get doublecolumn from application setting
            excelSerializer.DoubleColumn = Settings.Files.DoubleColumnFile
        End If

        excelSerializer.InformationLevel = Settings.Files.CompleteInfoFile

        Return excelSerializer
    End Function
End Class
